lib/OpenQA/Worker/Engines/isotovideo.pm

```diff
@@ -342,13 +343,14 @@ sub engine_workit {
     my $productdir  = OpenQA::Utils::productdir($vars{DISTRI}, $vars{VERSION}, $shared_cache);
     my $target_name = path($casedir)->basename;
 
-    $vars{ASSETDIR}   //= OpenQA::Utils::assetdir();
-    $vars{CASEDIR}    //= $target_name;
-    $vars{PRODUCTDIR} //= substr($productdir, rindex($casedir, $target_name));
+    $vars{ASSETDIR} //= OpenQA::Utils::assetdir();
+    $vars{CASEDIR}  //= $target_name;
 
-    if (!looks_like_url_with_scheme($vars{CASEDIR})) {
```

Comment:
I don't understand why you remove the `looks_like_url_with_scheme`-call again. If you remove it doesn't that also make the function unused?

Reply:
It's used in `engine_workit`

Reply:
If users define `CASEDIR=/tmp/xxx` or something else (not a github address), we can't do the symlink. Only the CASEDIR is the target name that we count, we could do the symlink. So I changed that check. This function `looks_like_url_with_scheme` is also used to check the `NEEDLES_DIR`, so I still keep it.